digits/dataset/job.py

```diff
@@ -31,6 +31,9 @@ def json_dict(self, verbose=False):
                                       "val_count":   t.val_count,
                                       "test_count":  t.test_count,
                                       } for t in self.parse_folder_tasks()],
+                'CreateDbTasks':    [{"db_name":            t.db_name,
+                                      "create_db_log_file": t.path(t.create_db_log_file),
+                                     } for t in self.create_db_tasks()],
```

Comment:
This opens an interesting can of worms. Do you envision exposing all available files via the REST API eventually?